modules/nw-sriov-rdma-example-mellanox.adoc

```diff
@@ -83,6 +84,8 @@ spec:
 =====
 See the `Configuring SR-IOV additional network` section for detailed explanation on each option in `SriovNetwork`.
 =====
++
+An optional library, app-netutil, provides several API methods for gathering network information about a pod from within a container running within that pod.
```

Comment:
Same

Reply:
The above note is existing text. However, since you are adding an additional information section, do you want to add a link to the the `Configuring SR-IOV additional network` section? It's a bit hard to find.

Reply:
A link would be great (and preferable) yet I understood from @jboxman that is it not possible to add a link in content modules